src/main/java/seedu/staffsnap/logic/parser/ApplicantBookParser.java

```diff
@@ -25,11 +26,15 @@
  */
 public class ApplicantBookParser {
 
+
     /**
      * Used for initial separation of command word and args.
      */
     private static final Pattern BASIC_COMMAND_FORMAT = Pattern.compile("(?<commandWord>\\S+)(?<arguments>.*)");
     private static final Logger logger = LogsCenter.getLogger(ApplicantBookParser.class);
+    private Boolean IsConfirmedNext = false;
+    private Boolean IsConfirmed = false;
```

Comment:
Perhaps follow the universal convention that class fields should be in camelCase? 🐫